catalog/designers/smartsheetinc/bio.html

```diff
@@ -0,0 +1,2 @@
+<p>Smartsheet Acquired Outfit.io - co-creators of the Outfit typeface</p>
```

Comment:
This is not a biography

Reply:
Yes, I was writing to the authors to modify it!